dvalin-tools/dvalin_tools/scrapers/events.py

```diff
@@ -149,8 +149,9 @@ def reparse_event_files(data_dir: Path) -> None:
         contents = event_file.read_text(encoding="utf-8")
         if contents.strip():
             existing_events = EventFile.model_validate_json(contents)
-            # for event in existing_events:
-            #     update_event_links_index(event)
+            for event in existing_events:
+                # update_event_links_index(event)
```

Comment:
Can this be removed?

Reply:
This is a method to reparse. Technically, I use it to have a fast method that will reparse all the files super quickly. Depending on what stage of the development I'm in, I put different things in this function, if I want it to also do some other types of operations before writing back to disk.

I agree that in a professional setting, I would also not like to push commented code.